mon-pix/app/components/tutorials/card.js

```diff
@@ -17,6 +18,10 @@ export default class Card extends Component {
     this.evaluationStatus = args.tutorial.isEvaluated ? buttonStatusTypes.recorded : buttonStatusTypes.unrecorded;
   }
 
+  get shouldShowActions() {
+    return !!this.currentUser.user;
```

Comment:
IMO ce genre de conditionnement n'est pas clair

Reply:
Tu préfères un `Boolean(this.currentUser.user)` ?

Reply:
C'est mieux mais je n'aime pas non plus. IMO `this.currentUser !== null` est plus clair mais y'a sûrement moyen de faire encore mieux ? (genre une méthode centralisée qui nous dit si un user est connecté ou non ?)

Reply:
Visiblement on a seulement besoin une seule fois de vérifier que l'`user` existe dans le reste de mon-pix et on fait comme la solution initiale que tu proposes donc go faire la même chose.

J'envoie un message à accès voir si quelque chose est prévu pour faire ça plus proprement.